projects/extension/Dockerfile

```
@@ -57,6 +57,19 @@ RUN mkdir -p /docker-entrypoint-initdb.d && \
     echo "echo \"shared_preload_libraries = 'timescaledb'\" >> \${PGDATA}/postgresql.conf" >> /docker-entrypoint-initdb.d/configure-timescaledb.sh && \
     chmod +x /docker-entrypoint-initdb.d/configure-timescaledb.sh
 
+###############################################################################
+# image for use in pgai-lib-db development
+FROM pgai-test-db as pgai-lib-db-dev
+ENV WHERE_AM_I=docker
+USER root
+
+RUN pip install --break-system-packages uv==0.6.3
+RUN mkdir /py/ && uv venv --directory /py/
```

This is missing correctly setting up dependencies in the container.
```suggestion
COPY pyproject.toml uv.lock /py/
RUN uv sync --directory /py --no-install-project --only-dev --frozen
```

---

yeah I do that in the just command docker-sync now. Let's review after merge.